codechallenge/pom.xml

```diff
@@ -0,0 +1,171 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <properties>
+        <docker.imageName>${artifactId}</docker.imageName>
+        <spring.version>4.1.5.RELEASE</spring.version>
+        <jersey.version>2.5</jersey.version>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.tradeshift</groupId>
+    <artifactId>codechallenge</artifactId>
+    <packaging>war</packaging>
+    <version>1.0-SNAPSHOT</version>
+    <name>codechallenge</name>
+
+    <build>
+        <finalName>codechallenge</finalName>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>2.5.1</version>
+                <inherited>true</inherited>
+                <configuration>
+                    <source>1.7</source>
+                    <target>1.7</target>
+                </configuration>
+            </plugin>
+
+
+            <plugin>
+                <groupId>com.atlassian.maven.plugins</groupId>
+                <artifactId>bash-maven-plugin</artifactId>
+                <version>1.0-ts-20141204</version>
+                <executions>
+                    <execution>
+                        <id>package</id>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <phase>package</phase>
+                        <configuration>
+                            <script>
+                                rsync -a --delete ${project.basedir}/docker/ ${project.build.directory}/docker
+                                cp -p ${project.build.directory}/${project.build.finalName}.war ${project.build.directory}/docker/root.war
+                                find ${project.build.directory}/docker | xargs touch -t 197409101630.00
+                                docker build -t tradeshift/${docker.imageName} ${project.build.directory}/docker
+                            </script>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>deploy</id>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <phase>deploy</phase>
+                        <configuration>
+                            <script>
+                                <!--docker push ${docker.imageName}-->
```

Comment:
I'm assuming that this is not supposed to be here? If not, this whole configuration block could be removed.